testdata/p4_16_samples_outputs/struct_init-frontend.p4

```diff
@@ -8,7 +8,7 @@ struct metadata_t {
 
 control I(inout metadata_t meta) {
     apply {
-        if (meta.foo == {9w192}) 
+        if (meta.foo == { 9w192 }) 
```

Comment:
I am worried about this change, this suggests that something is different about the type-checker.
I will try to find out what.